core/src/env_platform/resources/environment_platform.postman_collection.json

```
@@ -35,7 +35,62 @@
+					"raw": "http://0.0.0.0:9090/applications/tcf-20200513",
+					"protocol": "http",
+					"host": [
+						"0",
+						"0",
+						"0",
+						"0"
+					],
+					"port": "9090",
+					"path": [
+						"applications",
+						"tcf-20200513"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "putApplicationById",
+			"request": {
+				"method": "PUT",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n        \"status\": \"draft\",\n        \"title\": \"tree removal form\",\n        \"applicationCreatedDate\": {\n            \"year\": 4021,\n            \"month\": 2,\n            \"day\": 3,\n            \"hour\": 12,\n            \"minute\": 11\n        },\n        \"removalDate\": {\n            \"year\": 2020,\n            \"month\": 3,\n            \"day\": 1,\n            \"hour\": 12,\n            \"minute\": 0\n        },\n        \"reason\": \"for whatsss\",\n        \"applicationType\": \"Test\",\n        \"requestedBy\": \"Mr A R P Pieris\",\n        \"permitRequired\": true,\n        \"landOwner\": \"Mr A Wijedasa\",\n        \"treeRemovalAuthority\": \"Central\",\n        \"city\": \"Negombo\",\n        \"district\": \"Gampaha\",\n        \"nameOfTheLand\": \"Araliya Uyana\",\n        \"planNumber\": \"s12345\",\n        \"area\": [\n                {\"latitude\": 81.2343, \"longitude\": 76.2134},\n                {\"latitude\": 81.3243, \"longitude\": 76.2334},\n                {\"latitude\": 81.3343, \"longitude\": 76.2634}\n            ],\n        \"treeInformation\": [\n                {\n                    \"species\": \"kos\",\n                    \"treeNumber\": \"12\",\n                    \"heightType\": \"merchandise\",\n                    \"height\": 1230,\n                    \"girth\": 132,\n                    \"logDetails\": [\n                            {\"minGirth\": 120, \"maxGirth\": 132, \"height\": 600},\n                            {\"minGirth\": 110, \"maxGirth\": 120, \"height\": 590}\n                        ]\n                },\n                {\n                    \"species\": \"Jack\",\n                    \"treeNumber\": \"11\",\n                    \"heightType\": \"merchandise\",\n                    \"height\": 1230,\n                    \"girth\": 132,\n                    \"logDetails\": [\n                            {\"minGirth\": 222, \"maxGirth\": 532, \"height\": 340},\n                            {\"minGirth\": 110, \"maxGirth\": 230, \"height\": 670}\n                        ]\n                }\n            ]\n    }",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "http://0.0.0.0:9090/applications/tcf-20200513",
+					"protocol": "http",
+					"host": [
+						"0",
+						"0",
+						"0",
+						"0"
+					],
+					"port": "9090",
+					"path": [
+						"applications",
+						"tcf-20200513"
+					]
+				}
+			},
+			"response": []
 		}
 	],
 	"protocolProfileBehavior": {}
-}
+}
```

Add new line at end of file if there is no problem.
```suggestion
}

```